ui/app/components/job-status/panel/deploying.js

```diff
@@ -30,9 +30,7 @@ export default class JobStatusPanelDeployingComponent extends Component {
   // allocations we can track throughout a deployment
   establishOldAllocBlockIDs() {
     this.oldVersionAllocBlockIDs = this.job.allocations.filter(
-      (a) =>
-        a.clientStatus === 'running' &&
-        a.jobVersion !== this.deployment.get('versionNumber')
+      (a) => a.clientStatus === 'running' && a.isOld
```

Comment:
Side-effect: I found `isOld` for the first time and it'll help make a bunch of this more concise.